db/seeds.rb

```diff
@@ -27,11 +27,11 @@
     client.first_name = "#{user.profile.first_name}'s Client"
     client.last_name = 'a lastname'
     client.user = user
+    client.schedules << Schedule.build
   end
 end
 
-Volunteer.create(
-  first_name: 'First',
-  last_name: 'Volunteer',
-  email: 'volunteer@aoz.ch'
-)
+Volunteer.find_or_create_by!(first_name: 'First', last_name: 'Volunteer',
+  email: 'volunteer@example.com') do |volunteer|
```

Comment:
At first, I thought why you check for first_name, last_name and email? 

But then I realised that email isn't unique in volunteers.

I think it would make allot of sense to have unique email addresses for volunteers, and then afterwards this seed could only check for the email address.

in a migration:
```ruby
add_index :volunteers, :email, unique: true
```

**But in general**, _not referring to this pull request_:
I don't understand why the volunteers aren't a user role? 
This makes no sense to me!

The'll need to be able to log in and input data...